src/precommit-entry.js

```diff
@@ -42,6 +43,9 @@ export function precommit(path) {
   return getCommitMsg(path)
     .then(() => 0)
     .catch(err => {
+      console.log('Commit Message:');
+      console.log(fs.readFileSync(path, {encoding: 'utf8'}));
```

Comment:
This will be the second time reading this same file within this process. Perhaps you could carry over the first time you read it?